lib/heartbeat.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Watchman.Heartbeat do
+  use GenServer
+
+  def start_link(args) do
+    GenServer.start_link(__MODULE__, args, name: __MODULE__)
+  end
+
+  def init(args) do
+    spawn fn ->
+      send_heartbeat(args[:name], args[:interval])
+    end
```

Comment:
This looks ok, but maybe we can use a more OTP friendly solution. Take a looks at the following: http://stackoverflow.com/a/9431373.